app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -84,11 +94,37 @@ class AuditController @Inject() (
       conversionService.toJson(auditType.messageType.get, request.body)
     else EitherT.rightT(request.body)
 
-  private def getSource(auditType: AuditType, uri: Option[ObjectStoreResourceLocation], request: Request[Source[ByteString, _]])(implicit
+  private def fileId() =
+    s"{UUID.randomUUID().toString}-${DateTimeFormatter.ofPattern("yyyyMMdd-HHmmss.SSS").withZone(ZoneOffset.UTC).format(Instant.now())}"
```

Comment:
I think you forgot to put $. please use ```${UUID.randomUUID().toString}``` to generate the random UUID

Reply:
Yep - will fix :+1: